CHANGELOG.md

```diff
@@ -3,6 +3,20 @@
 Documentation for rocALUTION is available at
```

Comment:
This is the preferred wording:

Full documentation forrocALUTION is available at [https://rocm.docs.amd.com/projects/rocALUTION/en/latest/](https://rocm.docs.amd.com/projects/rocALUTION/en/latest/).